lib/archethic/beacon_chain/network_coordinates.ex

```diff
@@ -0,0 +1,193 @@
+defmodule Archethic.BeaconChain.NetworkCoordinates do
+  @moduledoc """
+  Network coordinates is a way to map latency between nodes and used to determine the closest nodes
```

Comment:
Perhaps you could provide more details on how to interpret the results and obtain the closest nodes

Reply:
This is already implemented here:

https://github.com/archethic-foundation/archethic-node/blob/39a8efde015bd7fedce84cd79decaefce911e18c/lib/archethic/p2p.ex#L348-L375

Reply:
ok! thank you for pointing that out 🙂
From my understanding of `GeoPatch`, we can have a third digit that corresponds to the precision. Do you think there is a way that would make sense to calculate the precision in the case of network latency? 🤔

Reply:
Yes it will be based on the bandwidth